hoot-core/src/main/cpp/hoot/core/io/OsmJsonWriter.cpp

```diff
@@ -57,17 +57,28 @@ HOOT_FACTORY_REGISTER(OsmMapWriter, OsmJsonWriter)
 
 OsmJsonWriter::OsmJsonWriter(int precision)
   : _includeDebug(ConfigOptions().getWriterIncludeDebugTags()),
-    _includeCompatibilityTags(true),
+    _includeCompatibilityTags(ConfigOptions().getJsonFormatHootenanny()),
     _precision(precision),
     _out(0),
     _pretty(ConfigOptions().getJsonPrettyPrint()),
     _writeEmptyTags(ConfigOptions().getJsonPerserveEmptyTags()),
-    _writeHootFormat(true),
+    _writeHootFormat(ConfigOptions().getJsonFormatHootenanny()),
```

Comment:
This little gem caused a bit of trouble because the default in `ConfigOptions.asciidoc` was `false` but here in the constructor it was set to `true`.  Not cool.  Fixed now.